operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -633,7 +630,9 @@ Product Managers rotate responsibility weekly for monitoring and responding to M
     </tr>
   </thead>
   <tbody></tbody>
-</table>\#\#\#\# Monthly Roadmap Q&A - Pre-sales and Support \| \| \| \| :--- \| :--- \|
+</table>
+
+### Monthly Roadmap Q&A - Pre-sales and Support
```

Comment:
This meeting is now: 
- Monthly Product:Sales Call
- 2nd Monday of the month, 9am to 10am PST 
- Attendees: All PMs; Optional: Sales, BizOps, Marketing departments 
- Format: 
  - PMs share roadmap highlights
    - Focus is on features with clear release timelines, and a clear benefit to customers, so Sales knows what is coming up soon they can talk to customers about 
    -  Demos, PRFAQs, and designs are shared as needed to illustrate the benefits
  - Open Q&A